ocp_resources/image_digest_mirror_set.py

```diff
@@ -0,0 +1,31 @@
+from ocp_resources.resource import Resource
+
+
+class ImageDigestMirrorSet(Resource):
+    """
+    API reference:
+    https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/imagedigestmirrorset-config-openshift-io-v1.html
+    """
+
+    api_group = Resource.ApiGroup.CONFIG_OPENSHIFT_IO
+
+    def __init__(self, image_digest_mirrors=None, **kwargs):
+        """
+        Create/Manage ImageDigestMirrorSet configuration object.
+
+        Args:
+            imageDigestMirrors (list of dict):
```

Comment:
```suggestion
            image_digest_mirrors (list of dict):
```
sorry, missed this before